RFCs/FS-1126-allow-lower-case-du-cases when-require-qualified-access-is specified.md

```diff
@@ -44,6 +47,26 @@ type DU = a
 type DU = | a
 
 type DU = | a
```

Comment:
We should explain the difference of these seemingly identical line, maybe with a comment that the top one doesn’t, and bottom one will raise an exception during compiling.